GreysonElkins/exquisite-corpse
# Feature/nh story setup refactor

#### What's this PR do?
- Refactors StoryEdit to be a functional component
- Moves all state info and relevant methods to StoryEditView
- Refactors the fetch call to use getData instead of getRandomPrompt
#### Where should the reviewer start?
StorySetupView
#### How should this be manually tested?
Click new story, choose a prompt, and start your story! A prompt from the appropriate genre should render if the user chooses to use one.
#### Any background context?
I'll revisit this to take care of ticket #35. The original plan was to write methods to utilize the fact that App holds all prompts, but I ran out of time so the original fetch call is still there.
#### What are the relevant tickets?
None yet!
#### Screenshots (if appropriate)
![cat](https://user-images.githubusercontent.com/60753853/93148007-64931e80-f6b8-11ea-8281-98af69d9c711.gif)
